### PR TITLE
Add KubeVirt GA documentation

### PR DESCRIPTION
**Pull request by mfranczy**
Signed-off-by: Marcin Franczyk <marcin0franczyk@gmail.com>

### CODE REVIEW

File: content/kubermatic/main/architecture/supported-providers/kubevirt/kubevirt.en.md [hunk 1]
```diff
@@ -49,748 +67,188 @@ spec:
...
-
 {{% notice warning %}}
-**Perform all the needed migration of your existing `MachineDeployment` following the below procedure according to each topic that needs migration.**
-Updating the `MachineDeployment` will create a new `VirtualMachine`  - Perform all the needed changes at once.
+Additionally consider [skipEvictionAfter](https://github.com/kubermatic/machine-controller/blob/main/cmd/machine-controller/main.go#L125-L126)
+parameter of Machine Controller that sets the timeout for workload eviction.  
+**Once exceeded, the VMs will simply be deleted.** 
 {{% /notice %}}
 
+### I discovered a DNS collision on my cluster. Why it happens?
```
Comment (hdurand0710) [inline content/kubermatic/main/architecture/supported-providers/kubevirt/kubevirt.en.md:212]: s/Why it happens/Why does it happen

Comment (mfranczy) [reply content/kubermatic/main/architecture/supported-providers/kubevirt/kubevirt.en.md]: done

**Comment by hdurand0710** [comment]
Good work! Looks pro.

**Comment by mfranczy** [comment]
thanks @hdurand0710, you have a typo in lgtm :), could you place a label again?

**Pull request merged**